Lib/pdb.py

```diff
@@ -2741,6 +2779,8 @@ def _ensure_valid_message(self, msg):
                 # Due to aliases this list is not static, but the client
                 # needs to know it for multi-line editing.
                 pass
+            case {"attach": int()}:
```

Comment:
Probably worth adding a comment explaining what this message type is used for, if only for consistency with the other message types.
```suggestion
            case {"attach": int()}:
                # Have the client to attach to a new process.
```